docs/learn/learn-nft-pallets.md

```diff
@@ -0,0 +1,101 @@
+functionalities that are not needed in the runtime.
+
+Some features are common to the NFT world, such as:
+
+- Buying an item up for sale.
+- Creating (i.e., mint) and burning (i.e., destroy) a collection of items or a single item (burning
+  must be signed either by the admin of the collection or the owner). Creating an item usually
+  involves setting some attributes specific to that item. Creating a collection also requires the
+  specification of the maximum number of items.
+
+  [Different settings](https://github.com/paritytech/substrate/pull/12483) are available for minting
+  collections:
+
+  - public mints: everyone can mint an asset (good for soul-bound collections)
+  - external mint: mint for holders of NFTs in another collection
+  - wave minting: mint X number of items that go to collection owners and Y number of items for the
+    public
+  - force mint: minting bypassing mint settings
+  - change max supply until it gets locked: possibility to change the supply for a limited amount of
+    times
+
+- [Smart attributes](https://github.com/paritytech/substrate/pull/12702) allow an NFT owner to grant
+  permission to other entities (another account, an application, etc.) to update attributes of an
+  NFT. An example could be that all Polkadot fellowship members have an NFT badge that gets updated
+  over time (sort of a rank) with a consequent upgrade in membership permissions.
+- A collection is managed by the Issuer, the Admin and the Freezer. Those roles can be changed
+  anytime, and there will be the option to attach
+  [multiple accounts per role](https://github.com/paritytech/substrate/pull/12437).
+- Setting metadata for an item or collection (metadata includes all essential information about the
+  item or the collection)
+- Setting or re-setting the price of an item.
+- Clearing attributes and metadata of a collection or an item.
+- Changing the owner of an item or a collection.
+- Transferring an item, as well as creating and canceling transfer approvals of a specific item, or
+  an [atomic swap](https://github.com/paritytech/substrate/pull/12285).
+- Transferring ownership of an item.
+
+:::info
+
+Metadata, attributes, and settings of an item or collection can be locked. The user can decide what
+to lock. Also, unauthorized and/or unprivileged transfers can be prevented by locking mechanism
+(unprivileged actions can be re-allowed anytime).
+
+:::
+
+Other features are less common and include:
+
+- Delegating accounts: delegated accounts can approve changes to an item's attributes and transfer
+  an item. The item owner always has control and can decide to cancel approvals from a delegated
+  account.
+- One can also execute pending atomic swaps created by a counterpart.
+
+### Work in Progress
+
+[NFTs fractionalization](https://github.com/paritytech/substrate/pull/12565) will allow the user to:
+
+- Take ownership of an NFT from the [pallet-uniques](#uniques-pallet)
```

Comment:
Should be: [pallet-nfts](#nfts-pallet)

Reply:
Updated! For the next PR, we will send it for your review before merging into the master.